include/exadg/incompressible_navier_stokes/spatial_discretization/operators/viscous_operator.h

```diff
@@ -77,7 +78,12 @@ class ViscousKernel
     if(data.viscosity_is_variable)
     {
       // allocate vectors for variable coefficients and initialize with constant viscosity
-      viscosity_coefficients.initialize(matrix_free, degree, data.viscosity);
+      viscosity_coefficients.initialize(matrix_free.n_cell_batches(),
+                                        matrix_free.n_inner_face_batches(),
+                                        matrix_free.n_boundary_face_batches(),
+                                        matrix_free.get_n_q_points(quad_index),
+                                        matrix_free.get_n_q_points_face(quad_index),
```

Comment:
Personally, I am not sure if this change is the right way. I am not sure if the old interface is the right one either. If we want to support mixed meshes, cells will have different number of quadrature points so that providing here a fixed value won't work.

Reply:
I see what you mean. The question is if we can keep using `dealii::Table` in that case as well. Maybe keeping the old interface is a good compromise right now to be able to adapt more easily later. What do you think?

Reply:
I brought back the old interface, of course now passing in `quad_index` alongside the `MatrixFree` object.